modules/modal/react/lib/Modal.tsx

```diff
@@ -50,17 +116,32 @@ export default class Modal extends React.Component<ModalProps> {
     open: false,
     padding: Modal.Padding.l,
     width: Modal.Width.s,
+    closeOnEscape: false,
```

Comment:
Any reason in particular that you picked `false` for the default here?

Reply:
Good question. Without a popup manager in Canvas Kit, I wanted to support it, but not force it on people. In very simple cases, this feature works, but it can do unexpected things when enabled. I figured defaulting to `false` was the safest bet.

Should we make this default to `true`?

Reply:
I think the majority of consumers would want it `true` by default. I would prefer to cater to the simpler use cases by default and provide advanced configuration if necessary